docs/contributors/code/getting-started-native-mobile.md

```diff
@@ -9,46 +9,46 @@ For a developer experience closer to the one the project maintainers current hav
 -   git
 -   [nvm](https://github.com/creationix/nvm)
 -   Node.js and npm (use nvm to install them)
--   [AndroidStudio](https://developer.android.com/studio/) to be able to compile the Android version of the app
+-   [Android Studio](https://developer.android.com/studio/) to be able to compile the Android version of the app
 -   [Xcode](https://developer.apple.com/xcode/) to be able to compile the iOS app
--   CocoaPods(`sudo gem install cocoapods`) needed to fetch React and third-party dependencies.
+-   CocoaPods (`sudo gem install cocoapods`) needed to fetch React and third-party dependencies.
 
 Note that the OS platform used by the maintainers is macOS but the tools and setup should be usable in other platforms too.
 
 ## Clone the project
 
-```
+```sh
```

Comment:
Both `sh` and `bash` language specifiers are used in documentation, but it seems like `sh` gives better results.